src/preamble.js

```diff
@@ -1221,12 +1218,19 @@ if (Module['buffer']) {
 #if BINARYEN
   if (typeof WebAssembly === 'object' && typeof WebAssembly.Memory === 'function') {
 #if ASSERTIONS
-    assert(TOTAL_MEMORY % WASM_PAGE_SIZE === 0);
```

Comment:
hmm, why did we remove the nice name? seems better than using the constant 65536 in multiple places